clientv3/balancer.go

```diff
@@ -66,6 +72,37 @@ func newSimpleBalancer(eps []string) *simpleBalancer {
 	return sb
 }
 
+func (b *simpleBalancer) getEndpoint(host string) string {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+
+	return b.host2ep[host]
+}
+
+func (b *simpleBalancer) updateHost2ep(eps []string, cs *credentials.TransportCredentials) {
```

Comment:
and it shouldn't need to take creds as an argument since it never actually uses it..